### PR TITLE
Added FHIR Data Model copy

### PR DESCRIPTION
**Pull request by kellytaylorgov**
@karlmdavis Can you take one more pass to verify the way we are providing an overview of the FHIR Data Model makes sense?

### CODE REVIEW

File: _pages/developers.md [hunk 1]
```diff
@@ -594,6 +597,38 @@ The above URL returns the synthetic beneficiary's Coverage information as an [Ex
...
+
+- [National Drug Code Directory](https://www.accessdata.fda.gov/scripts/cder/ndc/)
+- [HL7 v3 Code System ActCode](http://hl7.org/fhir/v3/ActCode/cs.html)
+- [ICD-10](http://hl7.org/fhir/sid/icd-10)
+
+[View the full list of Blue Button API FHIR Data Model Coding Systems and Identifiers](https://github.com/CMSgov/bluebutton-data-server/blob/master/dev/data-model.md)
+
+**Synthetic Data**
+
+For testing purposes, the CMS Blue Button API originally used a fake data set. This means that each claim—including its dates, codings, dollar amounts—was not based in reality, and that values were not associated with one another. For example, a claim for an annual checkup could have shown a $5,000 amount with a date of January 2020.
```
Comment (karlmdavis) [inline _pages/developers.md]: This paragraph still relevant/useful? If so, I'd at least add a hidden comment like this:

```
<<!-- TODO: Remove this history lesson anytime after 2018-06, as it won't be relevant anymore. -->
```

File: _pages/developers.md [hunk 2]
```diff
@@ -594,6 +597,38 @@ The above URL returns the synthetic beneficiary's Coverage information as an [Ex
...
+
+In December 2017, the CMS Blue Button API launched a new synthetic data set for developers to test against. This means that each claim returns a realistic value. For example, if a patient is prescribed the diabetes medication Metformin, the associated cost and date of this prescription will be accurate.
```
Comment (karlmdavis) [inline _pages/developers.md]: I'd also mention the following:

* The synthetic data only includes the following claim/event types: Carrier, Inpatient, Part D.
    * Not true of the production data.
* Some data fields are removed/rounded in the synthetic data, e.g. provider NPIs.
    * Not true of the production data.

File: index.md [hunk 3]
```diff
@@ -46,25 +46,23 @@ Participants in the Developer Preview can register an application and build inte
...
 
-The Blue Button API adds new value for beneficiaries, providers, care organizations and health applications, and supports a number of use cases within the areas of Healthcare and Life Sciences.
-
-#### You can use the Blue Button API to:
+Developers integrate with the Blue Button API adding value for beneficiaries, providers, care organizations, researchers and many more across Healthcare and Life Sciences to:
 
 **_Reduce patient burden_**
 
-- For example, a research organization can pre-populate medication lists for a patient during clinical trial enrollment
+A research organization can pre-populate a medication lists for a patient during clinical trial enrollment.
```
Comment (karlmdavis) [inline index.md]: Typo: "lists".

File: index.md [hunk 4]
```diff
@@ -46,25 +46,23 @@ Participants in the Developer Preview can register an application and build inte
 
 ## Value and Use Cases
 
-The Blue Button API adds new value for beneficiaries, providers, care organizations and health applications, and supports a number of use cases within the areas of Healthcare and Life Sciences.
-
-#### You can use the Blue Button API to:
+Developers integrate with the Blue Button API adding value for beneficiaries, providers, care organizations, researchers and many more across Healthcare and Life Sciences to:
```
Comment (karlmdavis) [inline index.md]: "Blue Button 2.0". Probably needs to be fixed all over the place.

File: _pages/developers.md [hunk 5]
```diff
@@ -594,6 +597,38 @@ The above URL returns the synthetic beneficiary's Coverage information as an [Ex
...
+- Part D Events
+- Skilled Nursing Facility Claims
+
+The Blue Button API FHIR data model leverages coding systems specific to Medicare billing forms and/or the Chronic Conditions Warehouse, FHIR and Industry Coding Systems.
+
+For Example:
+
+- [National Drug Code Directory](https://www.accessdata.fda.gov/scripts/cder/ndc/)
+- [HL7 v3 Code System ActCode](http://hl7.org/fhir/v3/ActCode/cs.html)
+- [ICD-10](http://hl7.org/fhir/sid/icd-10)
```
Comment (samgensburg-gov) [inline _pages/developers.md]: I think there's probably a better link for ICD 10. Maybe https://www.cdc.gov/nchs/icd/icd10cm.htm ?

File: _pages/developers.md (hunk 1 above)
Comment (samgensburg-gov) [inline _pages/developers.md]: I would remove this line. Most people never saw it the old way, so there's no reason to remind them.

File: _pages/developers.md [hunk 6]
```diff
@@ -594,6 +597,38 @@ The above URL returns the synthetic beneficiary's Coverage information as an [Ex
...
+
+**Synthetic Data**
+
+For testing purposes, the CMS Blue Button API originally used a fake data set. This means that each claim—including its dates, codings, dollar amounts—was not based in reality, and that values were not associated with one another. For example, a claim for an annual checkup could have shown a $5,000 amount with a date of January 2020.
+
+In December 2017, the CMS Blue Button API launched a new synthetic data set for developers to test against. This means that each claim returns a realistic value. For example, if a patient is prescribed the diabetes medication Metformin, the associated cost and date of this prescription will be accurate.
+
+Please note that this synthetic data set does not represent a longitudinal patient view. The claims—though representative independently—are shuffled and randomly assigned to patients. To build the synthetic data set, we selected a number of random claims, and shuffling them like a deck of cards among a group of fictitious Patient IDs. This will allow developers to test the Blue Button API system, but could result in a patient with records for contradictory procedures.
+
```
Comment (samgensburg-gov) [inline _pages/developers.md]: We need to add information about how to access this data. I'm happy to do it separately, but we need to document down to the level of usernames and passwords somewhere.

File: index.md (hunk 4 above)
Comment (kellytaylorgov) [reply index.md]: We will use Blue Button 2.0 in marketing materials but we have more flexibility in the application code and documentation.

**Pull request merged**